internal/rule/rule.go

```diff
@@ -19,6 +20,33 @@ type Rule struct {
 	Escalations      map[int64]*Escalation
 }
 
+// MarshalLogObject implements the zapcore.ObjectMarshaler interface.
+func (r *Rule) MarshalLogObject(encoder zapcore.ObjectEncoder) error {
+	encoder.AddInt64("id", r.ID)
+	encoder.AddString("name", r.Name)
+	// TODO: uncomment this once #188 is merged
+	// encoder.AddString("type", r.Type.String())
```

Comment:
If #188 adds a field that should be logged, that should be added as part of this PR, no need to add a TODO before. I doubt anyone searches the full code for TODO comments referring the current PR when reviewing it anyways.